### ISSUE TITLE
Is this still alive?

### ISSUE THREAD

**Issue by dzg** [neilalexander/seaglass#143]
This looks great ... is anyone still developing it? Seems to still be in Alpha. Thanks!

**Comment by keen99** [comment]
looks like it's a dead project - the website https://neilalexander.dev/seaglass points to a dead bintray for downloading releases, no commits since 2019.

:(

**Comment by neilalexander** [comment]
Yep, I'm afraid I am no longer maintaining Seaglass. I prioritised other personal projects instead but happy if anyone else wants to take over.
